Sources/MapboxNavigation/WayNameView.swift

```diff
@@ -10,10 +10,43 @@ open class WayNameLabel: StylableLabel {}
 /// :nodoc:
```

Comment:
If we expect the developer to use this class or any of its members, we should document it. Until now, these classes have largely been used in Style subclasses in a particular way, so they didn’t require much beyond an example in practice, though documentation is long overdue.

Reply:
Created ticket for this in https://github.com/mapbox/mapbox-navigation-ios/issues/3584.